concrete_datastore/api/v1/filters.py

```diff
@@ -203,8 +210,22 @@ def filter_queryset(self, request, queryset, view):
                 continue
             if param.replace('__in', '') not in filterset_fields:
                 continue
-
             values = query_params.get(param).split(',')
+            if isinstance(
+                queryset.model._meta.get_field(param.replace('__in', '')),
+                (UUIDField, ForeignKey),
+            ):
+                for value in values:
```

Comment:
I'd rather a utility function that check format of uuid

```python
def ensure_uuid_valid(value):
    try:
        uuid.UUID(value)
    except ValueError:
        return False
    else:
        return True
```

and check here

```python
for value un values:
    if not ensure_uuid_valid(value)
        raise ValidationError(
            {
                "message": (
                    f"{param}: '{value}' is not a valid UUID"
                )
            }
        )
````